src/fuck/Models/Command.cs

```diff
@@ -1,17 +1,18 @@
 ﻿using System.Collections.Generic;
-using System.Linq;
 
 namespace fuck.Models
 {
     public class Command
     {
-        public string OriginalCommand { get; set; }
-        public IEnumerable<string> UpdatedCommand => Recommendations.Select(x => Program + " " + x + " " + Arguments);

```

Comment:
Losing this will cause the program to not return any updated recommendations to run (unless you build up the list elsewhere).